int32        kp_yaw
int32        kp_pitch
int32        kp_depth
int32        kp_forward

int32        ki_yaw
int32        ki_pitch
int32        ki_depth
int32        ki_forward

int32        kd_yaw
int32        kd_pitch
int32        kd_depth
int32        kd_forward

int32       setpoint_yaw
int32       setpoint_pitch
int32       setpoint_depth
int32       setpoint_forward